corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -189,6 +189,33 @@ def test_process_doc_from_sql(self, datetime_mock):
 
         CaseAccessorSQL.hard_delete_cases(case.domain, [case.case_id])
 
+    @patch('corehq.apps.userreports.specs.datetime')
+    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+    def test_process_filter_no_longer_pass(self, datetime_mock):
+        datetime_mock.utcnow.return_value = self.fake_time_now
+        sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
+
+        self.pillow.process_change(doc_to_change(sample_doc))
+
+        sample_doc['type'] = 'wrong_type'
+
+        self.pillow.process_change(doc_to_change(sample_doc))
+        self.adapter.refresh_table()
+
+        self.assertEqual(0, self.adapter.get_query_object().count())
+
+    @patch('corehq.apps.userreports.specs.datetime')
+    @override_settings(TESTS_SHOULD_USE_SQL_BACKEND=True)
+    def test_check_if_doc_exist(self, datetime_mock):
+        datetime_mock.utcnow.return_value = self.fake_time_now
+        sample_doc, expected_indicators = get_sample_doc_and_indicators(self.fake_time_now)
+
+        self.assertFalse(self.adapter.doc_exists(sample_doc))
+
+        self.pillow.process_change(doc_to_change(sample_doc))
+
+        self.assertEqual(self.adapter.doc_exists(sample_doc), True)
```

Comment:
@snopoke I can't use in this place ```self.assertTrue``` because I want check in this line if the method returns ```True``` and nothing else. For ex: before I added ```scalar()```, method return us an object which means that ```self.assertTrue``` will have positive pass.